docs/_advanced/custom_routes.md

```diff
@@ -6,9 +6,43 @@ order: 10
 ---
 
 <div class="section-content">
+As of `v3.7.0`, custom HTTP routes can be easily added by passing in a collection of routes as `customRoutes` when initializing `App`. 
```

Comment:
```suggestion
As of `v3.7.0`, custom HTTP routes can be easily added by passing in an array of routes as `customRoutes` when initializing `App`. 
```

Reply:
Nit: We could be more specific here since keyed collections like Maps are not accepted.